specs.yml

```diff
@@ -1,10 +1,39 @@
 SAS:
-  EXPRESS:
-    LOG_MODE:
-      description: Log level
-      default: errors
-      regexp: ^error|all|none$
+  LOG:
+    CONSOLE_LEVEL:
+      description: Log level for console transport
+      default: info
+      regexp: ^error|warn|info|http|verbose|debug|silly$,
+
+    FILE_USE:
+      description: Wether or not to use file transport
+      default: yes
+      type: string
```

Comment:
This is probably where you should use a `boolean` instead of a string.
Cherry on top, `confmgr` will handle true/false as well as True/False, 0/1 and also yes/no.
Reference: https://gitlab.com/chevdor/confmgr/-/blob/master/src/ConfigManager.ts#L117